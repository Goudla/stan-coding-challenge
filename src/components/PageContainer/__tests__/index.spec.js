import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

// Scenes
import EntriesContainer from '../';

describe('<EntriesContainer />', () => {
  it('Should render', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<EntriesContainer>Test</EntriesContainer>);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
