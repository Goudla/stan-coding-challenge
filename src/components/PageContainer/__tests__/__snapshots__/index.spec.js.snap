// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EntriesContainer /> Should render 1`] = `
<Container>
  <div
    className="page-container"
  >
    Test
  </div>
</Container>
`;
